### PR TITLE
Add example systemd service file

### PR DESCRIPTION
**Pull request by svenseeberg**
I added an example for a systemd service file to run the daemon. This may be out of scope for your documentation or repo, but may be useful.

### CODE REVIEW

File: smbpasswd-web.service [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Change Passwords
```
Comment (0x3333) [inline smbpasswd-web.service]: Could you add a more descriptive description, like "Web interface to smbpasswd"

**Review by 0x3333 — CHANGES_REQUESTED** [review]
Great addition! Thanks!

File: smbpasswd-web.service (hunk 1 above)
Comment (svenseeberg) [reply smbpasswd-web.service]: Definitely valid ;-) I updated the commit.

Comment (0x3333) [reply smbpasswd-web.service]: Excelent! Thanks!

**Pull request merged**